project-koku/koku
# [COST-4459] Handle Subs variants and sending appropriate product ids

## Jira Ticket

[COST-4459](https://issues.redhat.com/browse/COST-4459)

## Description

This change will add in handling of HPC and SAP variants for subs processing. It creates a new column in the expected CSV output called `sap_bool` that evaluates to true or false and adjusts product ids for SAP/HPC variants according to the below:

SAP variants should include an additive product id based on the version of RHEL. 
- RHEL 7-: add '146' to the existing id's
- RHEL 8+: add '241' to the existing id's

HPC variants should specify the `Red Hat Enterprise Linux Compute` Role and send:
- RHEL 7-: use '76' as the product id
- RHEL 8+: use '479' as the product id

## Testing

1. Checkout Branch
2. Restart Koku
3. Load the test customer data
4. Check the CSV's in minio and see that the product id's for the appropriate resource-ids are still correct.

## Notes

...
